app/uk/gov/hmrc/exports/services/CountriesService.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services
+
+import play.api.libs.json._
+import uk.gov.hmrc.exports.models.Country
+
+object CountriesService {
+
+  private val countries: List[Country] = {
+    val jsonFile = getClass.getResourceAsStream("/code-lists/location-autocomplete-canonical-list.json")
+
+    def fromJsonFile: List[Country] =
+      Json.parse(jsonFile) match {
+        case JsArray(cs) =>
+          cs.toList.collect {
+            case JsArray(Seq(c: JsString, cc: JsString)) =>
+              Country(c.value, countryCode(cc.value))
+          }
+        case _ =>
+          throw new IllegalArgumentException("Could not read JSON array of countries from : " + jsonFile)
+      }
+
+    fromJsonFile.sortBy(_.countryName)
+  }
+
+  private def countryCode: String => String = cc => cc.split(":")(1).trim
+
+  val allCountries: List[Country] = countries
```

Comment:
i am not sure if this assignment val to val is good. 

Maybe we should provide just functions to find country by code ?